vslib/src/Sai.cpp

```diff
@@ -706,6 +706,23 @@ sai_status_t Sai::objectTypeGetAvailability(
             count);
 }
 
+sai_status_t Sai::queryAattributeCapability(
+        _In_ sai_object_id_t switch_id,
+        _In_ sai_object_type_t object_type,
+        _In_ sai_attr_id_t attr_id,
+        _Out_ sai_attr_capability_t *capability)
+{
+    MUTEX();
+    SWSS_LOG_ENTER();
+    VS_CHECK_API_INITIALIZED();
+
+    return m_meta->queryAattributeCapability(
```

Comment:
double "Aa" in name

Reply:
renamed the function name to remove the extra "a"